microsoft-edge/webview2/get-started/winui.md

```diff
@@ -51,195 +51,209 @@ Return here and continue with the steps below.
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*"/>
+            <ColumnDefinition Width="Auto"/>
+        </Grid.ColumnDefinitions>
+    
+        <TextBox Name="addressBar" Grid.Column="0"/>
+        <Button x:Name="myButton" Grid.Column="1" Click="myButton_Click">Go</Button>
+    
+        <controls:WebView2 x:Name="MyWebView"  Grid.Row="1" Grid.ColumnSpan="2"
+        Source="https://www.microsoft.com" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"/>
+    
+    </Grid>
+    ```
+    
+1.  In `MainWindow.xaml.cs`, copy the following code into `myButton_Click`.  This code navigates the WebView2 control to the URL entered in the address bar.
+
+    ```csharp
+    private void myButton_Click(object sender, RoutedEventArgs e)
+    {
+        try
+        {
+            Uri targetUri = new Uri(addressBar.Text);
+            MyWebView.Source = targetUri;
+        }
+        catch (FormatException ex)
+        {
+            // Incorrect address entered.
+        }
+    }
+    ```
+    
+1.  Select **File** > **Save All** (`Ctrl`+`Shift`+`S`) to save the project.
 
-   ```csharp
-   private void myButton_Click(object sender, RoutedEventArgs e)
-   {
-      try
-      {
-         Uri targetUri = new Uri(addressBar.Text);
-         MyWebView.Source = targetUri;
-      }
-      catch (FormatException ex)
-      {
-         // Incorrect address entered.
-      }
-   }
-   ```
+1.  Press **F5** to build and run the project.
 
-1. Select **File** > **Save All** (`Ctrl`+`Shift`+`S`) to save the project.
+1.  Enter a new complete URL in the address bar, such as **https://www.bing.com**, and then click the **Go** button.
 
-1. Press **F5** to build and run the project.
+    The WebView2 control in the sample app displays the Bing website.  The address bar displays the URL, such as `https://www.bing.com`:
 
-1. Enter a new URL in the address bar, and then choose **Go**.  For example, enter `https://www.bing.com`.
+    ![The sample app displays the Bing website](winui-images/getting-started-bing.png)
 
-   The sample app displays the Bing website. The address bar displays the URL https://www.bing.com:
+1.  Enter an incomplete URL in the address bar, such as `bing.com`, and then click the **Go** button.
 
-   ![The sample app displays the Bing website.](media/winui-getting-started-bing.png)
+    An `ArgumentException` exception is thrown,<!--and appears after you close the app--> because the URL doesn't start with `http://` or `https://`.
 
-1. Enter an incomplete URL in the address bar, such as `bing.com`.
+1.  Close the sample app.
 
-   An `ArgumentException` exception is thrown, because the URL doesn't start with `http://` or `https://`.
+1.  Close the sample app.  The following appears:
 
-1. Close the app.
+    ![Debugger not configured](winui-images/debugger-not-configd.png)
```

Comment:
@nishitha-burman - updated the article, except not getting the intended debugger behavior, shouldn't have to close the window to see that an exception was thrown, and shouldn't see this dialog.  Might try 2019 instead of VS 2022.

Reply:
noted current behavior in article